mitosheet/mitosheet/api/get_ai_completion.py

```diff
@@ -62,4 +62,7 @@ def get_ai_completion(params: Dict[str, Any], steps_manager: StepsManagerType) -
 
         # TODO: handle error here (e.g. it returns a 404, or something)
 
-        print(res.json())
\ No newline at end of file
+        print(res.json())
+        return json.dumps({
```

Comment:
While testing Snowflake, we realized that the best way to handle this is to use the API error message. Can we return that here as part / whole of the error?

Reply:
Sure thing!